Handle duplicate values in criteria detail fields in upload template (#596)
chpladmin/chpl-api: PENDING_PENDING_RELEASE_NOTES.md

## Version 20.1.0
_Date TBD_

### Bugs Fixed
* Save "Developer identified targeted users" on upload/confirm
* Update invalid error and warning messages for upload file

> ### New Features

Reviewer: Request putting this section above the Bugs section. Usually have the sections in order of major-minor-bug

> * When validating pending listings, check for duplicate values, remove the duplicate value, and provide a warning message that the duplicate was removed

Reviewer: ```suggestion
 * When validating pending listings, check for duplicate values, remove the duplicate value(s), and provide a warning message about the removal
```
